galasa-parent/dev.galasa/src/main/java/dev/galasa/api/run/Run.java

```diff
@@ -24,15 +22,15 @@ public class Run {
     private String  stream;
     private String  repo;
     private String  obr;
-    private Boolean local;
-    private Boolean trace;
+    private boolean local;
```

Comment:
isLocal ?
isTraceEnabled ?

Reply:
Done, changed their getters to follow conventions, however left the old versions in with a comment as this is likely exposed and used code by clients.